integration-tests/api/__tests__/admin/product.js

```diff
@@ -1435,38 +1435,39 @@ describe("/admin/products", () => {
   })
 
   describe("DELETE /admin/products/:id/options/:option_id", () => {
+    const adminHeaders = {
```

Comment:
comment: ahah I was more thinking bellow the timeout in order to be able to use it in the entire file as it is always the same